homeassistant/components/hyperion/light.py

```diff
@@ -47,24 +47,29 @@
 
 SUPPORT_HYPERION = SUPPORT_COLOR | SUPPORT_BRIGHTNESS | SUPPORT_EFFECT
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Required(CONF_HOST): cv.string,
-        vol.Required(CONF_PORT, default=DEFAULT_PORT): cv.port,
-        vol.Optional(CONF_DEFAULT_COLOR, default=DEFAULT_COLOR): vol.All(
-            list,
-            vol.Length(min=3, max=3),
-            [vol.All(vol.Coerce(int), vol.Range(min=0, max=255))],
-        ),
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-        vol.Optional(CONF_PRIORITY, default=DEFAULT_PRIORITY): cv.positive_int,
-        vol.Optional(
-            CONF_HDMI_PRIORITY, default=DEFAULT_HDMI_PRIORITY
-        ): cv.positive_int,
-        vol.Optional(CONF_EFFECT_LIST, default=DEFAULT_EFFECT_LIST): vol.All(
-            cv.ensure_list, [cv.string]
-        ),
-    }
+PLATFORM_SCHEMA = vol.All(
+    cv.deprecated(CONF_HDMI_PRIORITY, invalidation_version="0.116"),
+    cv.deprecated(CONF_DEFAULT_COLOR, invalidation_version="0.116"),
+    cv.deprecated(CONF_EFFECT_LIST, invalidation_version="0.116"),
```

Comment:
Please have two releases for the deprecation time.
```suggestion
    cv.deprecated(CONF_HDMI_PRIORITY, invalidation_version="0.118"),
    cv.deprecated(CONF_DEFAULT_COLOR, invalidation_version="0.118"),
    cv.deprecated(CONF_EFFECT_LIST, invalidation_version="0.118"),
```